config/locales/en.yml

```diff
@@ -2062,7 +2111,56 @@ en:
       Request data
     data_request_sent: >
       We have received your request and will send you your data as soon as possible.
-    redeem_voucher: Redeeem Voucher
+    redeem_voucher: Redeem Voucher
```

Comment:
This might be a very contrived example that might not be worth to consider, just wanted to point it out:

- User verifies valid voucher code
- User selects valid tutorial
- Teacher removes tutorial
- User clicky on "redeem voucher". They get the message that their tutor status was updated successfully even though it wasn't as the tutorial didn't exist anymore at that time. This is not a security issue, it's just that the user might be confused after this message that they can't access any tutorials. But to be honest: what's the probability of this happening? I guess too low to bother...

Reply:
As I wrote above: The tutor status was still upgraded in the sense that the user can now be selected by the teacher for tutorials, so it is a "tutor without tutorial".

Reply:
Ah ok, yeah that makes sense. I've reworked the messages a bit in the latest commits.